contracts/src/v0.8/dev/ocr2dr/OCR2DRClient.sol

```diff
@@ -33,10 +34,20 @@ abstract contract OCR2DRClient is OCR2DRClientInterface {
    * @notice Sends OCR2DR request to the stored oracle address
    * @param req The initialized OCR2DR.Request
    * @param subscriptionId The subscription ID
+   * @param gasLimit gas limit for the fulfillment callback
+   * @param confirmations  How many blocks you'd like the DON to wait before responding to the request.
    * @return requestId The generated request ID
    */
-  function sendRequest(OCR2DR.Request memory req, uint256 subscriptionId) internal returns (bytes32) {
-    bytes32 requestId = s_oracle.sendRequest(subscriptionId, OCR2DR.encodeCBOR(req));
+  function sendRequest(
+    OCR2DR.Request memory req,
+    uint64 subscriptionId,
+    uint32 gasLimit,
+    uint8 confirmations
```

Comment:
On some networks like polygon we can have thousands of blocks for a confirmation... Let's not limit this to uint8, maybe uint32 would be safe enough.